### PR TITLE
[Service Fabric] Change all enum values to readable strings

### PR DESCRIPTION
**Pull request by msjeffreychen**

### CODE REVIEW

File: arm-servicefabric/2016-01-28/swagger/servicefabric.json [hunk 1]
```diff
@@ -2974,7 +2974,7 @@
     "EventHealthEvaluation": {
       "type": "object",
       "allOf": [{"$ref": "#/definitions/HealthEvaluation"}],
-      "x-ms-discriminator-value": "1",
+      "x-ms-discriminator-value": "EventHealthEvaluation",
```
Comment (amarzavery) [inline arm-servicefabric/2016-01-28/swagger/servicefabric.json]: what is the reason behind this change?

Comment (msjeffreychen) [inline arm-servicefabric/2016-01-28/swagger/servicefabric.json]: We decided to use a readable string, so user could read the type without the need to map it.

Comment (amarzavery) [inline arm-servicefabric/2016-01-28/swagger/servicefabric.json]: Oh awesome. I just wanted to be sure that this has not been done due to any shortcoming from the AutoRest side. Btw, I like the approach of using a readable string v/s a number.

Comment (amarzavery) [inline arm-servicefabric/2016-01-28/swagger/servicefabric.json]: Wait. As per the documentation for ["x-ms-discriminator-value"](https://github.com/Azure/autorest/blob/master/Documentation/swagger-extensions.md#x-ms-discriminator-value), it is used for specifying the value on wire. If your value on the wire is same as the model name then why do you need this extension ?

Comment (msjeffreychen) [inline arm-servicefabric/2016-01-28/swagger/servicefabric.json]: I have few ones that the model name isn't exactly the same as the wire name. Do I just need "x-ms-discriminator-value" for those ones, or do I need for all in this case?

Comment (amarzavery) [inline arm-servicefabric/2016-01-28/swagger/servicefabric.json]: Yup. You should only have the extension for those that have a different name on the wire

Comment (msjeffreychen) [inline arm-servicefabric/2016-01-28/swagger/servicefabric.json]: I have fixed the spec. Thanks.
